src/sentry/models/actor.py

```diff
@@ -93,6 +107,22 @@ def get_actor_identifier(self):
         return self.get_actor_tuple().get_actor_identifier()
 
 
+def get_actor_id_for_user(user: Union["User", RpcUser]):
+    # Handy for JIT creation of user actors
+    if user.actor_id:
+        return user.actor_id
+    # Temporary Dual write
+    actor = Actor.objects.filter(type=1, user_id=user.id).first()
+    if actor is None:
+        actor = Actor.objects.create(
```

Comment:
Should we catch an integrity error here and skip?  Like so this remains an idempotent "put".

Reply:
Added a fix for this -- short answer is that if the user does not have the actor_id, we always try to create the actor through.